web/locales/en/messages.ftl

```diff
@@ -8,6 +8,8 @@ email-input =
 submit-form-action = Submit
 loading = Loading…
 email-opt-in-info = I'd like to receive emails such as goal reminders, my progress updates and newsletters about Common Voice.
+email-opt-in-info-title = Join the Common Voice mailing list
+email-opt-in-info-sub = Receive emails such as goal reminders, progress updates and newsletters about Common Voice.
```

Comment:
I'm still learning how the localization system works. As you can see, this is _very_ similar to the `email-opt-in-info` string above. When an edit like this is needed, if I update the initial value will changes eventually propagate to new locales? Or is it necessary to create a new ID like I did here?

Reply:
Unfortunately we do need new IDs when we want the localization community to notice. So you did the right thing!